cmd/scheduler/entrypoints/precheck.go

```diff
@@ -0,0 +1,47 @@
+package entrypoints
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/avast/retry-go"
+	adminClient "github.com/flyteorg/flyteidl/clients/go/admin"
+	"github.com/pkg/errors"
+
+	"github.com/spf13/cobra"
+)
+
+var preCheckRunCmd = &cobra.Command{
+	Use:   "precheck",
+	Short: "This command will check pre requirement for scheduler",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := context.Background()
+		err := retry.Do(
+			func() error {
+				config := adminClient.GetConfig(ctx)
+				host := strings.Split(config.Endpoint.Host, ":")
+				response, err := http.Get(fmt.Sprintf("http://%s/healthcheck", host))
```

Comment:
Isn't there a grpc call to do the health check

Reply:
Yes added grpc call